### ISSUE TITLE
Illuminate/console versions not supported for laravel 8

### ISSUE THREAD

**Issue by ParogDev** [SocialiteProviders/Generators#29]
I'm very thankful for your package, it made my life a lot easier. 

The package illuminate/console hasn't been updated yet for Laravel 8, the upgrade to Laravel 8 fails with your package as a dependency.

I'm hoping people can continue using it with Laravel 8 in the near future!

**Comment by the-provost** [comment]
Encountered the same issue. It is irreplaceable in my project.

**Comment by ParogDev** [comment]
> Encountered the same issue. It is irreplaceable in my project.

@the-provost I'm curious as to what project you have going that uses an oauth provider creation tool. I was able to continue mine just by removing the package from the dependencies and removing the service and still use the custom providers it created.

**Comment by the-provost** [comment]
@ParogDev An SSO project. Its an internal app for my company. I have created a PR for the issue.  Fingers crossed.

**Comment by jeremykenedy** [comment]
I use this package and to work around the PR needed: 
1. Fork this package and make the patch.
2. Add a version/release a release to the fork.
3. I call the original package in my `composer.json`, 
4. I link out to the [fork](https://github.com/jeremykenedy/Generators-1) which has the patch. You can use your own fork or mine.
5. Update the original composer.json calling the package to the version number.
6. Add the map in your project's composer.json

And it works. 

Here is an example of what I mean taken from [my auth repo](https://github.com/jeremykenedy/laravel-auth/blob/master/composer.json).

```
{
    ...

    "require": {

        ...

        "socialiteproviders/generators": "^4.7",

        ...
    },

    ...

    "repositories": [
        {
            "type": "vcs",
            "url":  "https://github.com/jeremykenedy/Generators-1.git"
        }
    ],

    ...

}

**Comment by the-provost** [comment]
Thanks @jeremykennedy . I was thinking of doing the same too

**Comment by m1guelpf** [comment]
Closed by #30
